client/src/components/Leaflet.js

```diff
@@ -129,14 +131,14 @@ const Leaflet = ({
           marker.bindPopup(m.name)
         }
         if (m.onMove) {
-          marker.on("move", event => m.onMove(event, map))
+          marker.on("moveend", event => m.onMove(event, map))
```

Comment:
This is intentional and mandatory. In order to be able keep new **Latitude, Longitude** form fields in sync with map, we re-initialize marker layer on the map whenever makers passed to map changes (i.e. lat, lng fields of a marker). We cannot re-initialize marker layer in the middle of dragging because we are already dragging a marker within that layer. `moveend` seemed to me as the easiest and optimal solution for this case.